warehouse/query-core/src/main/java/datawave/query/tables/ShardQueryLogic.java

```diff
@@ -1344,7 +1395,163 @@ public void close() {
                 log.error("Caught exception trying to close Scheduler", e);
             }
         }
+    }
+
+    @Override
+    public Object validateQuery(AccumuloClient client, Query settings, Set<Authorizations> auths, boolean expandFields, boolean expandValues) throws Exception {
+        this.config = ShardQueryConfiguration.create(this, settings);
+        if (log.isTraceEnabled()) {
+            log.trace("Initializing ShardQueryLogic for query validation: " + System.identityHashCode(this) + '('
+                            + (this.getSettings() == null ? "empty" : this.getSettings().getId()) + ')');
+        }
+
+        // todo - maybe unnecessary, we could just return early if no rules configured.
+        if (validationRules == null || validationRules.isEmpty()) {
+            throw new IllegalStateException("Query validation rules not configured.");
```

Comment:
I would think in this case that query validation is not supported.  Call super.validateQuery instead to throw the sane UnsupportedOperationException.